staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -43,5 +43,5 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
+	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.GA},
```

Comment:
I searched the codebase, and can't find any references to it other than here, line 35 above, and line 106 of `pkg/features/kube_features.go`